core/src/main/java/bisq/core/provider/fee/FeeService.java

```diff
@@ -51,42 +54,72 @@
 // TODO use dao parameters for fee
 @Slf4j
 public class FeeService {
+
+    ///////////////////////////////////////////////////////////////////////////////////////////
+    // Static
+    ///////////////////////////////////////////////////////////////////////////////////////////
+
     // Miner fees are between 1-600 sat/byte. We try to stay on the safe side. BTC_DEFAULT_TX_FEE is only used if our
     // fee service would not deliver data.
     private static final long BTC_DEFAULT_TX_FEE = 50;
+    private static final long MIN_PAUSE_BETWEEN_REQUESTS_IN_MIN = 2;
+    private static BsqStateService bsqStateService;
+    private static PeriodService periodService;
 
-    private static long MIN_MAKER_FEE_BTC = 5_000; // 0.005%. 0.5 USD at BTC price 10_000 USD;
-    private static long MIN_TAKER_FEE_BTC = 5_000;
-    private static long DEFAULT_MAKER_FEE_BTC = 200_000; // 0.2%. 20 USD at BTC price 10000 USD for a 1 BTC trade;
-    private static long DEFAULT_TAKER_FEE_BTC = 200_000;
+    private static long getFeeFromParam(Param parm) {
+        return bsqStateService != null && periodService != null ? bsqStateService.getParamValue(parm, periodService.getChainHeight()) : 0;
+    }
 
-    // 0.05 BSQ (5 satoshi) for a 1 BTC trade -> 0.005%. 0.05 USD if 1 BSQ = 1 USD, 10 % of BTC fee
-    private static final long MIN_MAKER_FEE_BSQ = 5;
-    private static final long MIN_TAKER_FEE_BSQ = 5;
-    // 2 BSQ or 200 BSQ-satoshi. About 2 USD if 1 BSQ = 1 USD for a 1 BTC trade which is about 10% of a normal BTC fee.
-    private static final long DEFAULT_TAKER_FEE_BSQ = 200;
-    private static final long DEFAULT_MAKER_FEE_BSQ = 200;
+    public static Coin getMakerFeePerBtc(boolean currencyForFeeIsBtc) {
```

Comment:
This is a bit confusing. Probably better to have separate methods for the two coins. If necessary to have just one method it should be clear what the Coin returned might be.

Reply:
Hm.. the handling of the BTC or BSQ fee is not very nice atm... would require a bit more refactoring at the clients side... Atm I don't want to touch more as it is needed so prefer to keep it but yes, its an open TODO...